pages/common/aws-cognito-idp.md

```diff
@@ -0,0 +1,28 @@
+# aws cognito-idp
+
+> Manage Amazon Cognito user pool and its users and groups using the CLI.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cognito-idp/index.html>.
+
+- Create a new Cognito user pool:
+
+`aws cognito-idp create-user-pool --pool-name {{name}}`
+
+- List all user pools:
+
+`aws cognito-idp list-user-pools --max-results 10`
```

Comment:
```suggestion
`aws cognito-idp list-user-pools --max-items {{10}}`
```
I couldn't find `max-results` in the man page, instead I found `max-items`. 
[Reference](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cognito-idp/list-user-pools.html)

Reply:
The doc is wrong they have not updated it.
![image](https://user-images.githubusercontent.com/23727670/195988598-2bb42135-f0b4-4c47-8fce-a1d05e062c29.png)

I always test commands before putting it here.

Reply:
> The doc is wrong they have not updated it. !

Thanks for the information.

> I always test commands before putting it here.

That's a great thing keep up the good work. [Big fan of you on LinkedIn]

Then my updated suggestion would be just to include curly braces for the value

```suggestion
`aws cognito-idp list-user-pools --max-results {{10}}`
```

Reply:
Glad to hear that @kbdharun I have made it variable, hope you can approve now.